Frontend/ui-customer/src/Flow.purs

```diff
@@ -351,6 +351,7 @@ enterMobileNumberScreenFlow = do
             setValueToLocalStore MOBILE_NUMBER (state.data.mobileNumber)
             setValueToLocalStore COUNTRY_CODE (state.data.countryObj.countryCode)
             void $ liftFlowBT $ setCleverTapUserData "Phone" (state.data.countryObj.countryCode <> (getValueToLocalStore MOBILE_NUMBER))
+            void $ pure $ setCleverTapUserProp [{key : "Mobile Number", value : unsafeToForeign $ state.data.countryObj.countryCode <> (getValueToLocalStore MOBILE_NUMBER)}]
```

Comment:
reduce usage of `getValueToLocalStore` we can get the same value from `state.data.mobileNumber`